parsers/PE.py

```diff
@@ -67,7 +67,7 @@ def fetch_production(country_code='PE', session=None):
             data[i]['production'][MAP_GENERATION[k]] = \
                 data[i]['production'].get(MAP_GENERATION[k], 0) + v['Valor'] / interval_hours
 
-    return filter(lambda x: validate(x) is not None, data)
+    return list(filter(lambda x: validate(x) is not None, data))
```

Comment:
This might be a bit cleaner as `return [x for x in data if validate(x)]`

Reply:
Agreed I think that's a lot cleaner.  Once we've got everything moved to Python 3 we can refactor some of these fixes.